### PR TITLE
[TACHYON-999] Fix formatting in BlockMasterSync

### PR DESCRIPTION
**Pull request by phvu**
JIRA: https://tachyon.atlassian.net/browse/TACHYON-999

### CODE REVIEW

File: servers/src/main/java/tachyon/worker/block/BlockMasterSync.java [hunk 1]
```diff
@@ -35,39 +35,57 @@
 /**
  * Task that carries out the necessary block worker to master communications, including register and
  * heartbeat. This class manages its own {@link tachyon.client.WorkerBlockMasterClient}.
- *
+ * <p/>
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockMasterSync.java]: I don't think the license should use this tag.

File: servers/src/main/java/tachyon/worker/block/BlockMasterSync.java [hunk 2]
```diff
@@ -35,39 +35,57 @@
...
  * When running, this task first requests a block report from the
  * {@link tachyon.worker.block.BlockDataManager}, then sends it to the master. The master may
  * respond to the heartbeat with a command which will be executed. After which, the task will wait
  * for the elapsed time since its last heartbeat has reached the heartbeat interval. Then the cycle
  * will continue.
- *
+ * <p/>
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockMasterSync.java]: I don't think the license should use this tag.

File: servers/src/main/java/tachyon/worker/block/BlockMasterSync.java [hunk 3]
```diff
@@ -35,39 +35,57 @@
...
  * If the task fails to heartbeat to the master, it will destroy its old master client and recreate
  * it before retrying.
  */
 public final class BlockMasterSync implements Runnable {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   private static final int DEFAULT_BLOCK_REMOVER_POOL_SIZE = 10;
-  /** Block data manager responsible for interacting with Tachyon and UFS storage */
+  /**
+   * Block data manager responsible for interacting with Tachyon and UFS storage
+   */
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockMasterSync.java]: For all of these variables, if the javadoc fits on one line, we should just keep the original formatting.

File: servers/src/main/java/tachyon/worker/block/BlockMasterSync.java [hunk 4]
```diff
@@ -120,8 +136,7 @@ public void registerWithMaster() {
    * Main loop for the sync, continuously heartbeats to the master node about the change in the
    * worker's managed space.
    */
-  @Override
-  public void run() {
+  @Override public void run() {
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockMasterSync.java]: I think Tachyon code places override on its own line, like before.

File: servers/src/main/java/tachyon/worker/block/BlockMasterSync.java [hunk 5]
```diff
@@ -225,8 +240,7 @@ public BlockRemover(BlockDataManager blockDataManager, long sessionId, long bloc
       mBlockId = blockId;
     }
 
-    @Override
-    public void run() {
+    @Override public void run() {
```
Comment (gpang) [inline servers/src/main/java/tachyon/worker/block/BlockMasterSync.java]: I think Tachyon code places override on its own line, like before.

**Comment by gpang** [comment]
I added some comments. Thanks for this improvement!

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by phvu** [comment]
Ok I will fix it.

**Comment by phvu** [comment]
then actually this doesn't change anything much :)

**Comment by jsimsa** [comment]
LGTM +1

**Comment by gpang** [comment]
LGTM

**Pull request merged**